### ISSUE TITLE
Dependency conflict with Laravel 9

### ISSUE THREAD

**Issue by WillyReyno** [jikan-me/jikan#395]
Hello, I was in the process of upgrading to Laravel 9, and got the following conflict while trying to install `v3.0.0-rc.3` : 

``` 
  Problem 1
    - fabpot/goutte v4.0.1 requires symfony/mime ^4.4|^5.0 -> found symfony/mime[v4.4.0-BETA1, ..., 4.4.x-dev, v5.0.0-BETA1, ..., 5.4.x-dev] but these were not loaded, likely because it conflicts with another require.
    - jikan-me/jikan v3.0.0-rc.3 requires fabpot/goutte 4.0.1 -> satisfiable by fabpot/goutte[v4.0.1].
    - Root composer.json requires jikan-me/jikan v3.0.0-rc.3 -> satisfiable by jikan-me/jikan[v3.0.0-rc.3].
```

This should be fixed with fabpot/goutte v4.0.2, but I remember you mentioned there was a [breaking change in it]( https://github.com/jikan-me/jikan/issues/379#issuecomment-1001671103), but I hope you can find a workaround to let people upgrade to L9.

Thanks for the awesome work as always!

**Comment by ToshY** [comment]
@irfan-dahir Came across the same issue when updating from `v3.0.0-beta.9` to `v3.0.0-rc.3` in my Symfony project. Forcing the upgrade with all dependencies I see the following downgrades occurring.

```
Lock file operations: 0 installs, 7 updates, 0 removals
  - Downgrading fabpot/goutte (v4.0.2 => v4.0.1)
  - Upgrading jikan-me/jikan (v3.0.0-beta.9 => v3.0.0-rc.3)
  - Downgrading symfony/mime (v6.0.7 => v5.4.7)
  - Downgrading symfony/property-access (v6.0.7 => v5.4.7)
  - Downgrading symfony/property-info (v6.0.7 => v5.4.7)
  - Downgrading symfony/serializer (v6.0.7 => v5.4.7)
  - Downgrading symfony/string (v6.0.3 => v5.4.3)
```

I don't know what the exact usage of `parents()` is (like stated in this [issue](https://github.com/jikan-me/jikan/issues/379)), but wouldn't replacing it with `ancestors()` (like stated in these [release notes](https://github.com/symfony/dom-crawler/blob/e6f132c117f50858e2d0a6ea8919796241be4a18/CHANGELOG.md)) solve the problem?

**Comment by emre-bozkurt** [comment]
is there any solution for the problem same here i can't install the package on Laravel 9 and PHP 8.

**Comment by irfan-dahir** [comment]
Resolved with #405 Special thanks to @rjd22 for the PR.
Tag: https://github.com/jikan-me/jikan/releases/tag/v3.0.0-rc.5
Tested with Lumen ^9 ✅ 
PHP: ^8 is now minimum

Feel free to re-open the issue if there's a problem.

**Comment by irfan-dahir** [comment]
Resolved with #405 Special thanks to @rjd22 for the PR.
Tag: https://github.com/jikan-me/jikan/releases/tag/v3.0.0-rc.5
Tested with Lumen ^9 ✅ 
PHP: ^8 is now minimum

Feel free to re-open the issue if there's a problem.
